src/doc/trpl/concurrency.md

```diff
@@ -135,28 +135,25 @@ This gives us an error:
         ^~~~
 ```
 
-In this case, we know that our code _should_ be safe, but Rust isn't sure. And
-it's actually not safe: if we had a reference to `data` in each thread, and the
-thread takes ownership of the reference, we have three owners! That's bad. We
-can fix this by using the `Arc<T>` type, which is an atomic reference counted
-pointer. The 'atomic' part means that it's safe to share across threads.
```

Comment:
I've updated the pull request. How's it look now?

I originally had a full exposition on Reference counting and Atomic operations but decided to strip it out for brevity's sake since "Atomic Reference Count" is mentioned at the top of the file in the explanation of Sync.

However I can definitely see how it's useful to have it explained near the code that uses it.